task/dev13/src/test/java/MailRuChromeTest.java

```diff
@@ -0,0 +1,95 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class MailRuChromeTest {
+
+  private WebDriver driver;
+
+  @Before
+  public void setUp() throws Exception {
+    System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+    driver = new ChromeDriver();
+    driver.get("http://www.mail.ru");
+  }
+
+  @Test
+  public void correctPasswordTest() {
+    WebElement login = driver.findElement(By.id("mailbox__login"));
+    login.sendKeys("tat-dev13");
+    WebElement pasword = driver.findElement(By.id("mailbox__password"));
+    pasword.sendKeys("23.03.2017");
+    driver.findElement(By.id("mailbox__auth__button")).click();
+    driver.findElement(By.className("b-datalist__body"));
```

Comment:
This is not test, nothing is checked (asserts, expected exceptions are not used)